### ISSUE TITLE
output is the same

### ISSUE THREAD

**Issue by LemonLov** [NVIDIA-AI-IOT/torch2trt#542]
when I was trying to convert hrnet, it gives me so many warning like:
![54b85e546ef139886b57ef9809567b6](https://user-images.githubusercontent.com/26343220/114256477-08913e00-99ec-11eb-9459-f22018d47e73.png)

And when predicting, it says [TensorRT] ERROR: INVALID_ARGUMENT: Can not find binding of given name.  And the predict result is the same. 

My environment as follows:
pytorch 1.4.0, cudatoolkit 10.1.243, TensorRT 6.0.1.5

Hope for you reply. Thanks.

**Comment by jaybdub** [comment]
Hi @LemonLov ,

Thanks for reaching out!

These warnings indicate potential layers which don't yet have converters implemented.  

One option is to attempt to resolve these by adding custom converters.  You may find the following helpful

The batch_norm converter is currently implemented, but is filtered for TensorRT 7+.

https://github.com/NVIDIA-AI-IOT/torch2trt/blob/master/torch2trt/converters/batch_norm.py

A version compatible with TensorRT 6 is here https://github.com/NVIDIA-AI-IOT/torch2trt/blob/master/torch2trt/converters/BatchNorm2d.py, but hasn't been registered with ``torch.batch_norm`` or ``torch.nn.functional.batch_norm`` yet.

https://github.com/NVIDIA-AI-IOT/torch2trt/blob/master/torch2trt/converters/batch_norm.py

It's still possible other issues persist.  Do you mind sharing a link to the HRNet model you're using?  In case you run into trouble, I can track this model to see if support can be implemented.

Best,
John
